manifests/f/FxSoundLLC/FxSound/1.1.26.0/FxSoundLLC.FxSound.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with komac v2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: FxSoundLLC.FxSound
+PackageVersion: 1.1.26.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /exenoui
+  SilentWithProgress: /exenoui
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/fxsound2/fxsound-app/raw/14145b12e8da6ef6e381649ca0fab23584043eff/release/fxsound_setup.exe

```

```suggestion
  InstallerUrl: https://github.com/fxsound2/fxsound-app/raw/v1.1.26.0/release/fxsound_setup.exe
```

Using the release's tag appears to work.